web/src/main/webapp/my-app/layout/services.js

```diff
@@ -79,27 +80,21 @@ define(['angular', 'jquery'], function(angular, $) {
                 method: 'POST',
                 url: SERVICE_LOC.newLayout,
                 data: {"layout" : newLayout, "new" : false},
-                dataType: 'json',
-                headers : {
-                  'eppn': 'pnogal@wisc.edu',
-                  'Content-Type': 'application/x-www-form-urlencoded'
-                }
+                dataType: 'json'
               });
-            })
+            });
```

Comment:
What's up with a semicolon addition here? (I haven't looked closely, just flagging it because I didn't expect it. If it's correct, cool, merge away. And if it's not correct, merge away anyway if need be; if it's important we can figure it out in predev. :)